stacks/OrgDisruptionsGenerator.ts

```diff
@@ -1,14 +1,19 @@
 import { TreatMissingData } from "aws-cdk-lib/aws-cloudwatch";
 import { SnsAction } from "aws-cdk-lib/aws-cloudwatch-actions";
+import { SubnetType } from "aws-cdk-lib/aws-ec2";
 import { PolicyStatement } from "aws-cdk-lib/aws-iam";
 import { Function, StackContext, use } from "sst/constructs";
 import { DynamoDBStack } from "./DynamoDBStack";
 import { MonitoringStack } from "./MonitoringStack";
+import { RdsStack } from "./RdsStack";
+import { VpcStack } from "./VpcStack";
 import { createBucket } from "./utils";
 
 export const OrgDisruptionsGeneratorStack = ({ stack }: StackContext) => {
     const { disruptionsTable } = use(DynamoDBStack);
     const { alarmTopic } = use(MonitoringStack);
+    const { dbUsernameSecret, dbPasswordSecret, dbNameSecret, dbHostROSecret, dbPortSecret } = use(RdsStack);
```

Comment:
I left it in for now just in case we need to fall back to using it. It's still quite a heavy query since it needs to call the ref data api to get the service centre points for each service. I'd prefer to either store the centre points in the json blob in rds or move the reference data into the same db and just query it directly.

Reply:
Actually scratch that, I'll just get rid, we can always restore it from version control. I'll get rid of the SIRI-SX importer as well while I'm at it